src/style.css

```diff
@@ -1,3 +1,146 @@
-body {
+html {
+  --hover-color: #00acdc;
+}
+
+* {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, system-ui;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  box-sizing: border-box;
```

Comment:
it's better to define in * selector

And vice versa: padding, margin shouldn't be defined in * selector. Reset it for the list of tags you really need